explorer/app/components/Header/components/Logo/logo.tsx

```diff
@@ -0,0 +1,33 @@
+// Core dependencies
+import { StaticImageData } from "next/image";
+import Link from "next/link";
+import React from "react";
+
+// App dependencies
+import { StaticImage } from "../../../StaticImage/StaticImage";
+
+export interface LogoProps {
+  alt: string;
+  height?: number;
+  imgSrc: StaticImageData;
+  link: string;
+  width?: number;
+}
+
+export const Logo = ({
+  alt,
+  height,
+  imgSrc,
```

Comment:
What are your thoughts on updating this to just `src`. The other attributes use a standard HTML attribute naming convention and maybe we can update this one to match?

Reply:
Done